src/query/api/v1/middleware/metrics.go

```diff
@@ -59,8 +65,10 @@ func ResponseMetrics(iOpts instrument.Options, route *mux.Route) mux.MiddlewareF
 				path = "unknown"
 			}
 
+			querySize := calculateQuerySize(w, r, cfg)
```

Comment:
I wonder if it also makes sense to emit the range? Like, if we change the threshold in config, that could reclassify what's considered large and what's considered small which would make past results less useful?

Reply:
Ah yeah, good call; will add this and also emit the series count threshold too :+1: